hack/install.sh

```diff
@@ -78,6 +78,10 @@ echo_docker_as_nonroot() {
 
 	Remember that you will have to log out and back in for this to take effect!
 
+	WARNING: adding non-root user to "docker" group, by default, will grant the
+	         ability to run privileged containers, which can be used to obtain
```

Comment:
Maybe we should leave out `privileged` here, because running `-v /:/host` also allows root access to the filesystem; including access to the docker socket (to start a privileged container); no need to have a `--privileged` container in the first place